doc/spinel-protocol-src/spinel-basis-detail-status.md

```diff
@@ -0,0 +1,55 @@
+  1         | `STATUS_FAILURE`                  | Operation has failed for some undefined reason.
+  2         | `STATUS_UNIMPLEMENTED`            | The given operation has not been implemented.
+  3         | `STATUS_INVALID_ARGUMENT`         | An argument to the given operation is invalid.
+  4         | `STATUS_INVALID_STATE`            | The given operation is invalid for the current state of the device.
+  5         | `STATUS_INVALID_COMMAND`          | The given command is not recognized.
+  6         | `STATUS_INVALID_INTERFACE`        | The given network link identifier is not supported.
+  7         | `STATUS_INTERNAL_ERROR`           | An internal runtime error has occurred.
+  8         | `STATUS_SECURITY_ERROR`           | A security or authentication error has occurred.
+  9         | `STATUS_PARSE_ERROR`              | An error has occurred while parsing the command.
+ 10         | `STATUS_IN_PROGRESS`              | An error has occurred while parsing the command.
+ 11         | `STATUS_NOMEM`                    | The operation has been prevented due to memory pressure.
+ 12         | `STATUS_BUSY`                     | The device is currently performing a mutually exclusive operation.
+ 13         | `STATUS_PROP_NOT_FOUND`           | The given property is not recognized.
+ 14         | `STATUS_PACKET_DROPPED`           | The packet was dropped.
+ 15         | `STATUS_EMPTY`                    | The result of the operation is empty.
+ 16         | `STATUS_CMD_TOO_BIG`              | The command was too large to fit in the internal buffer.
+ 17         | `STATUS_NO_ACK`                   | The packet was not acknowledged.
+ 18         | `STATUS_CCA_FAILURE`              | The packet was not sent due to a CCA failure.
+ 19         | `STATUS_ALREADY`                  | The operation is already in progress, or the property already has the value.
+ 20         | `STATUS_ITEM_NOT_FOUND`           | The given item could not be found in the property. 
+ 21         | `STATUS_INVALID_COMMAND_FOR_PROP` | The given command cannot be performed on this property.
+112         | `STATUS_RESET_POWER_ON`           | Cold power-on start.
+113         | `STATUS_RESET_EXTERNAL`           | External device reset.
+114         | `STATUS_RESET_SOFTWARE`           | Internal software orderly reset.
+115         | `STATUS_RESET_FAULT`              | Internal software abortive reset.
+116         | `STATUS_RESET_CRASH`              | Unrecoverable software execution failure.
+117         | `STATUS_RESET_ASSERT`             | Software invariant property not respected.
+118         | `STATUS_RESET_OTHER`              | Unspecified cause.
+119         | `STATUS_RESET_UNKNOWN`            | Failure while recovering cause of reset.
+120         | `STATUS_RESET_WATCHDOG`           | Software failed to make sufficient progress.
+
+EDITOR: The `STATUS_CCA_FAILURE` status is a technology-specific status code. Should not be in the basis protocol.
```

Comment:
`STATUS_CCA_FAILURE` doesn't seem technology-specific to me. You can have a CCA failure on any half-duplex medium.